Sources/Rendering/OpenGL/Texture/index.js

```diff
@@ -1365,11 +1366,13 @@ function vtkOpenGLTexture(publicAPI, model) {
     // WebGL2 path, we have 3d textures etc
     if (model._openGLRenderWindow.getWebgl2()) {
       if (
-        dataType === VtkDataTypes.FLOAT ||
-        (model.useHalfFloat &&
-          (dataType === VtkDataTypes.SHORT ||
-            dataType === VtkDataTypes.UNSIGNED_SHORT))
```

Comment:
@finetjul the indent was removed from upstream here. I agree it is a bit strange but seems like it is following the ESLint.